sdk/python/kfp/components/component_decorator_test.py

```diff
@@ -149,4 +149,4 @@ def comp(text: str) -> List[str]:
 
         # TODO: ideally should be the canonical type string, rather than the specific annotation as string, but both work
         self.assertEqual(comp.component_spec.outputs['Output'].type,
-                         'typing.List[str]')
+                         'typing.List[str]')
```

Comment:
nit: remove this diff

When we go back and look at this PR to recall what changes were made, it will be helpful to not have superfluous changes that distract from the true content.